README.md

```diff
@@ -11,28 +11,36 @@ This sample requires the following:
 
 ## Register the application 
  
-1. Navigate to the [the Azure portal - App registrations](https://go.microsoft.com/fwlink/?linkid=2083908) to register your app. Login using a **Work or School Account**. 
+1. Determine your ASP.NET app's URL. In Visual Studio's Solution Explorer, select the Microsoft-Graph-ExcelRest-ToDo project. In the Properties window, find the value of SSL URL. Copy this value.  
+![Application URL](images/ApplicationURL.PNG) 
+
+2. Navigate to the [the Azure portal - App registrations](https://go.microsoft.com/fwlink/?linkid=2083908) to register your app. Login using a **Work or School Account**.  
+![New registration Page](images/NewRegistration.PNG)
  
-2. Select **New registration**. On the **Register an application** page, set the values as follows. 
+3. Select **New registration**. On the **Register an application** page, set the values as follows. 
  
 * Set **Name** to Microsoft-Graph-ExcelRest-ToDo. 
 * Set **Supported account types** to **Accounts in any organizational directory**. 
 * Leave **Redirect URI** empty. 
-* Choose **Register**. 
- 
-3. On the **Microsoft-Graph-ExcelRest-ToDo** page, copy and save the values for the **Application (client) ID** and the **Directory (tenant) ID**. You will need them in step 7. 
+* Choose **Register**.  
+![Application Registration](images/ApplicationRegistration.PNG)
  
-4. Select **Certificates & secrets** under **Manage**. Select the **New client secret** button. Enter a value in **Description**, select any option for **Expires** and choose **Add**. 
+4. On the **Microsoft-Graph-ExcelRest-ToDo** page, copy and save the values for the **Application (client) ID**.  
+![Application Registration](images/ApplicationID.PNG) 
  
-5. Copy the client secret value before leaving the page. You will need it in the next step. 
+5. Select **Certificates & secrets** under **Manage**. Select the **New client secret** button. Enter a value in **Description**, select any option for **Expires** and choose **Add**.  
+![Certificates & secrets](images/ClientSecretRegistration.PNG)
  
-6. Open the sample solution in Visual Studio and then open the **Constants.cs** file. Change the **Tenant** string to the **Directory (tenant) ID** value you copied earlier. Similarly, change the **ClientIdForAppAuthn** string to the **Application (client) ID** value and change the **ClientSecret** string to the client secret value. 
+6. Copy the client secret value before leaving the page. You will need it in the next step.  
+ ![Certificates & secrets](images/ClientSecret.PNG)
  
-7. Return to the Azure Active Directory management center. Select **API permissions** and then select **Add a permission**. On the panel that appears, choose **Microsoft Graph** and then choose **Application permissions**. 
+7. Select **API permissions** and then select **Add a permission**. On the panel that appears, choose **Microsoft Graph** and then choose **Application permissions**.  
```

Comment:
There is no need for this step as scopes are defined explicitly in the config file. See https://github.com/microsoftgraph/aspnet-todo-rest-sample/blob/249e3f6b898ce7bcd3accc5086a9098317f943c5/Microsoft-Graph-ExcelRest-ToDo/Web.config#L14

Reply:
Okay, I have deleted the steps.